src/utils/index.ts

```diff
@@ -8,3 +10,9 @@ export function assertIsValidKeyGetter<R>(keyGetter: unknown): asserts keyGetter
     throw new Error('Please specify the rowKeyGetter prop to use selection');
   }
 }
+
+export function getCellStyle<R, SR>(column: CalculatedColumn<R, SR>): React.CSSProperties {
+  return column.frozen
+    ? { left: `var(--frozen-left-${column.key})` }
+    : { gridColumnStart: column.idx + 1 };
```

Comment:
This "picks" the correct grid column from the `grid-template-columns`, regardless of its DOM position.